ui/objects/templates/semantic-ui/objects/Detail.jinja

```diff
@@ -1,359 +1,308 @@
+              <p>
+                Technologie: {% for val in value.fabricationTechnologies %}
+                {{ val.title }}
+                {% endfor %}
+              </p>
+              {% endif %}
             </div>
-            {% endif %}
-    
-    
-    
-          <!-- desc -->
-          {% if metadata.restorationObject.description %}
-          <div class="details__div__desc" aria-label="Popis předmětu">
-            <p class="parag">{{ metadata.restorationObject.description[0].value }}</p>
           </div>
-          {% endif %}
-    
-    
-          <div class="vr-huge-horiz"></div>
-    
-    
+
+          {% endfor %}
+        </div>
+        {% endif %}
+
+        <!-- desc -->
+        {% if metadata.restorationObject.description %}
+        <div aria-label="Popis předmětu">
+          <p>{{ metadata.restorationObject.description[0].value }}</p>
+        </div>
+        {% endif %}
+
+        <div class="ui divider"></div>
+
         <!-- info table-->
         {% if metadata.restorationWork.abstract %}
-            <div class="ui  two column divided grid details__div__info" aria-label="Info o předmětu">
-        {% else %}
-            <div class="ui  one column  grid details__div__info" aria-label="Info o předmětu">
-        {% endif %}
-            <div class="column details__div__info-left">
-    
-            <table summary="Informace o předmětu">
-              <tbody>
-                {% if metadata.restorationObject.restorationRequestor %}
-                <tr class="details__div__info-left__box">
-                  <td class="parag">Zadavatel</td>
-                  <td class="parag">
-                    {{ metadata.restorationObject.restorationRequestor.title }}
-                  </td>
-                </tr>
-                {% endif %} {% if metadata.restorationWork.restorer or
-                metadata.restorationWork.workType %}
-                <tr class="details__div__info-left__box">
-                  <td class="parag">Restauroval(a)</td>
-                  <td class="parag">
-                    {% if metadata.restorationWork.restorer %}
-                    {{ metadata.restorationWork.restorer }}
-                    {% endif %} {% if metadata.restorationWork.workType %} ,
-                    {{ metadata.restorationWork.workType.title }}
-                    {% endif %}
-                  </td>
-                </tr>
-                {% endif %} {% if metadata.restorationWork.supervisors %}
-                <tr class="details__div__info-left__box">
-                  <td class="parag">Vedoucí práce</td>
-    
-                  <td class=" vert-div parag">
-                    {% for value in metadata.restorationWork.supervisors %}
-                    <div class="horiz-div parag">
-                         {{ value.fullName }}
+        <div class="ui two column divided grid" aria-label="Info o předmětu">
+          {% else %}
+          <div class="ui one column grid" aria-label="Info o předmětu">
```

Comment:
aria-label is a duplication of one at table, remove here

Reply:
What happened to the aria-label? it seems to have completely disappeared, even on that table

Reply:
Initially there was a dependency on existence of description of the object. If there is no description then there will be one column grid. In the case if there should be aria-label I should return this to the both of them (it will be the same as was commented as duplication)

Reply:
Still an issue